engine/class_modules/sc_hunter.cpp

```diff
@@ -1863,6 +1872,16 @@ struct kill_command_bm_t: public kill_command_base_t
 
     return am;
   }
+
+  double composite_crit_chance() const override
+  {
+    double cc = kill_command_base_t::composite_crit_chance();
+
+    if ( o()->options.t28_2pc )
+      cc *= 1 + (0.15 * p()->buffs.frenzy->check());
```

Comment:
This should be `+=` as crit chance increases are typically additive.

Reply:
Fixed